templates/components/entry.html.twig

```diff
@@ -15,7 +15,7 @@
     {% elseif entry.cross %}
         {{ component('entry_cross', {entry: entry}) }}
     {% else %}
-        <article{{ attributes.defaults({
+        <thread{{ attributes.defaults({
```

Comment:
I don't think that `thread` is a valid html5 element, `article` is: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Reply:
ow haha, that could be right yes.